src/lib.rs

```diff
@@ -18,6 +18,9 @@
 //!   typically the most efficient.
 //!   - [Parallel iterators][iter module] make it easy to convert a sequential iterator to
 //!     execute in parallel.
+//!     - The [`ParallelIterator`] trait defines general methods for for all parallel iterators.
```

Comment:
I'm going to sneak in a quick typo fix:

```suggestion
//!     - The [`ParallelIterator`] trait defines general methods for all parallel iterators.
```

Reply:
Seems your PR is not open to maintainer pushes. I might just push this after bors is done.

Reply:
The check-box for maintainers to push is checked, though. Nevertheless, thanks, I've accepted the suggestion. But it seems that have canceled bors :-(.

Reply:
Maybe it's just the web UI that doesn't work for that? Anyway, thank for applying, and I've reapproved the PR.